Sources/NetworkProtection/VPNConfiguration/VPNConfigurationManager.swift

```
@@ -23,14 +23,12 @@ public final class VPNConfigurationManager {
 
     public init() {}
 
-    public func removeVPNConfiguration() async {
-        let tunnels = try? await NETunnelProviderManager.loadAllFromPreferences()
+    public func removeVPNConfiguration() async throws {
```

No longer silencing errors.  We surface them.